repo-structure.md

```diff
@@ -0,0 +1,169 @@
+|   |   └── common-pipeline-n/
+|   |           └── [pipeline files that make up a full tekton pipeline used with all collections in incubator category]
+├── collection-1/
+|   ├── [collection files - see collection structure below]
+├── collection-2/
+|   └── [collection files - see collection structure below]
+stable
+├── common/
+|   ├── pipelines/
+|   |   ├── common-pipeline-1/
+|   |   |       └── [pipeline files that make up a full tekton pipeline used with all collections in stable category]
+|   |   └── common-pipeline-n/
+|   |           └── [pipeline files that make up a full tekton pipeline used with all collections in stable category]
+├── collection-1/
+|   ├── [collection files - see collection structure below]
+├── collection-n/
+|   └── [collection files - see collection structure below]
+experimental
+├── common/
+|   ├── pipelines/
+|   |   ├── common-pipeline-1/
+|   |   |       └── [pipeline files that make up a full tekton pipeline used with all collections in experimental category]
+|   |   └── common-pipeline-n/
+|   |           └── [pipeline files that make up a full tekton pipeline used with all collections in experimental category]
+├── collection-1/
+|   ├── [collection files - see collection structure below]
+└── collection-n/
+    └── [collection files - see collection structure below]
+```
+
+## Collection structure
+
+There is a standard structure that all collections follow. The structure below represents the source structure of a collection:
+
+```bash
+my-collection
+├── README.md
+├── stack.yaml
+├── collection.yaml
+├── image/
+|   ├── config/
+|   |   └── app-deploy.yaml
+|   ├── project/
+|   |   ├── [files that provide the technology components of the stack]
+|   |   └── Dockerfile
+│   ├── Dockerfile-stack
+|   └── LICENSE
+├── pipelines/
+|   ├── my-pipeline-1/
+|   |       └── [pipeline files that make up the full tekton pipeline]
+└── templates/
+    ├── my-template-1/
+    |       └── [example files as a starter for the application, e.g. "hello world"]
+    └── my-template-2/
+            └── [example files as a starter for a more complex application]
+
+```
+
+The structure above is then processed when you build the collection, to generate a Docker image for the stack, along with tar files of each of the templates and pipelines, which can then all be stored/referenced in a local or public appsody repo. Refer to the section on [Building and Testing Stacks](/docs/stacks/build-and-test.md) for more details. The appsody CLI can then access such a repo, to use the stack to initiate local development.
+
+## Summary of files within the stack and collection source and user directory structure
+
+### stack.yaml
+
+The `stack.yaml` file in the top level directory defines the different attributes of the stack and which template the stack should use by default. See the example below:
+
+```bash
```

Comment:
should use `yaml` highlighting here instead of bash

Reply:
This was copied from some Appsody .md files but happy to change. Done.